src/chapters/aspdotnet-model-validation/exercises.rst

```diff
@@ -0,0 +1,42 @@
+Exercises: ViewModels and Model Validation
+==========================================
+
+Let’s practice adding more properties to our event objects and 
+validating them. Create a new branch from the `display-error-messages branch <https://github.com/LaunchCodeEducation/CodingEventsDemo/tree/display-error-messages>`__. 
+
+Below, we describe some new properties for you to add to the ``Event`` class. 
+For each property, consider the following factors:
+
+#. What will you call your property?
+#. What type of input should be added to capture the field's information from the user?
+#. Refer to the chapter content to find an appropriate attributes to fit the necessary constraints. 
```

Comment:
Grammar: "...to find appropriate attributes..."

Reply:
Fixed!